.github/workflows/_build.yml

```diff
@@ -191,8 +208,12 @@ jobs:
             adb logcat > logcat.log &
             ./gradlew :app:record${{ env.flavour }}${{ env.config }}AndroidTestScreenshotTest -Pandroid.testInstrumentationRunnerArguments.variant=${{ matrix.variant }}
 
-            changedFiles=`git diff --name-only` && echo "::set-output name=CHANGED_FILES::${changedFiles//$'\n'/'%0A'}"
-            ./scripts/check-no-changes.sh
+            changedFiles=`git diff --name-only` && echo "CHANGED_FILES=${changedFiles//$'\n'/'%0A'}" >> $GITHUB_OUTPUT
+
+      - name: Check snapshot changes
```

Comment:
this script will fail the build when there's changes - we only want that if we're not updating snapshots, so moved to a separate step